test/factory/common_factory_tests.dart

```diff
@@ -39,6 +39,11 @@ void commonFactoryTests(Function factory) {
       var instance = factory({}, ['one', 'two',]);
       expect(getJsChildren(instance), equals(['one', 'two']));
     });
+
+    test('an empty List', () {
+      var instance = factory({}, []);
+      expect(getJsChildren(instance), equals([]));
+    });
```

Comment:
Could you add one more test that a single iterable child is converted to a list successfully? (A regression test for your fix in 3dce4a543a113cd86d50774b7ddcbef79f79fec8)